app/src/main/java/nerd/tuxmobil/fahrplan/congress/settings/widgets/SwitchPreference.kt

```diff
@@ -0,0 +1,63 @@
+package nerd.tuxmobil.fahrplan.congress.settings.widgets
+
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.selection.toggleable
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.unit.dp
+import nerd.tuxmobil.fahrplan.congress.designsystem.buttons.Switch
+import nerd.tuxmobil.fahrplan.congress.designsystem.indicators.ripple
+import nerd.tuxmobil.fahrplan.congress.designsystem.modifiers.WithoutRipple
+import nerd.tuxmobil.fahrplan.congress.designsystem.modifiers.minimumInteractiveComponentSize
+
+@Composable
+internal fun SwitchPreference(
+    title: String,
+    checked: Boolean,
+    onCheckedChange: (Boolean) -> Unit,
+    modifier: Modifier = Modifier,
+    subtitle: String? = null,
+) {
+    val interactionSource = remember { MutableInteractionSource() }
+
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
```

Comment:
:nail_care: static import for `CenterVertically`

Reply:
Done.